setup.py

```diff
@@ -42,12 +50,7 @@
         'aiodocker',
     ],
     python_requires='>=3.5',
-    install_requires=[
-        'setuptools==36.2.0',
-        'aiohttp==2.2.3',
-        'yarl>=0.10',
-    ],
-    extras_require={
-        'test': ['pytest', 'pytest-asyncio', 'flake8'],
-    }
+    install_requires=requirements,
```

Comment:
Nevertheless the lib work with some versions but not all. For aiohttp I suppose we are following semantic versioning so maybe we could use `<2.3`. What do you think would be best? Simply removing any version restriction?